cms/djangoapps/contentstore/views/course.py

```diff
@@ -334,9 +341,6 @@ def course_rerun_handler(request, course_key_string):
     GET
         html: return html page with form to rerun a course for the given course id
     """
-    # Only global staff (PMs) are able to rerun courses during the soft launch
-    if not GlobalStaff().has_user(request.user):
```

Comment:
This disallowed any non-staff user to re-run a course so I removed it. A condition in this method already checks for access of course so a learner or a course creator who has no staff/instructor rights for a course will not be able to re-run a course.

Reply:
Also, kindly provide feedback if this condition should be added with the waffle switch `enable_global_course_creator`?